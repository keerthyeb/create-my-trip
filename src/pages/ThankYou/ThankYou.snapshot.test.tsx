import { render } from '@testing-library/react';
import TripCreatedPage from './ThankYou';

test('TripCreatedPage component snapshot', () => {
  const { container } = render(<TripCreatedPage />);
  expect(container).toMatchSnapshot();
});
