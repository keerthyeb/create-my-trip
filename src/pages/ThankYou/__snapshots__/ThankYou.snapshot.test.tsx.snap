// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TripCreatedPage component snapshot 1`] = `
<div>
  <div
    class="css-1r3ldka"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        style="margin-top: 17%;"
      />
      ;
      <div
        style="min-height: 35vh; padding: 40px; margin: 10px; display: flex; background: white; border-radius: 10px; justify-content: space-evenly; flex-direction: column; box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
          style="font-family: cursive;"
        >
          Thank you for inquiring about your trip of a lifetime with us. Our team of experts will be in touch soon to help craft the perfect vacation for you.
        </h4>
        <div>
          <h5
            class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            style="font-family: cursive;"
          >
            Sincerely,
          </h5>
          <h5
            class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            style="font-family: cursive;"
          >
            Enchanting Travels
          </h5>
        </div>
      </div>
    </div>
  </div>
</div>
`;
